### PR TITLE
CE-3634 Don't show when loading page into VE

### PR DESCRIPTION
**Pull request by kamilkoterba**
@Wikia/spitfires

### CODE REVIEW

**Comment by kamilkoterba** [comment]
Tested LGTM

File: extensions/wikia/SpitfiresContributionExperiments/scripts/my-profile-tour-config.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+define('myProfileTourConfig', [], function () {
+	'use strict';
+
+	return [
+		
+	];
+});
```
Comment (dianafa) [inline extensions/wikia/SpitfiresContributionExperiments/scripts/my-profile-tour-config.js]: should this be part of this PR? where is this used?

File: extensions/wikia/SpitfiresContributionExperiments/templates/MyProfileTourExperimentStep.mustache [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<div class="my-profile-step {{stepClass}}"></div>
+{{#shouldShowBack}}<a class="my-profile-header-go-back">BACK</a>{{/shouldShowBack}}
```
Comment (dianafa) [inline extensions/wikia/SpitfiresContributionExperiments/templates/MyProfileTourExperimentStep.mustache]: should `BACK` be hardcoded?

File: extensions/wikia/SpitfiresContributionExperiments/scripts/my-profile-tour-config.js (hunk 1 above)
Comment (kamilkoterba) [inline extensions/wikia/SpitfiresContributionExperiments/scripts/my-profile-tour-config.js]: You saw nothing ;)

**Comment by Grunny** [comment]
LGTM.

**Pull request merged**